Vagrantfile

```diff
@@ -0,0 +1,20 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Build/rebuild documentation : vagrant up
```

Comment:
I saw this now. Please add this description to the [writing documentation section](https://s-enda-documentation.readthedocs.io/en/issue55_central_system/architecture.html) as well. Are the html files placed in the same folder as in the first alternative (i.e., `./build/html/devel_environ.html`)?

Reply:
Refactored write_docs.rst, and added a section for how to build documentation with Vagrant.